debian/ubuntu-advantage-tools.postinst

```diff
@@ -253,12 +293,11 @@ case "$1" in
       # machine-access cache files no longer present or used since 20.1
       rm -f /var/lib/ubuntu-advantage/private/machine-access-*.json
 
-      if [ "14.04" = "$VERSION_ID" ]  || [ "16.04" = "$VERSION_ID" ]; then
+      if check_release_supports_esm "${UBUNTU_CODENAME}"; then
```

Comment:
This is a great improvement that will hopefully mean we need to update this less for each ubuntu release.